Automattic/jetpack
# Search: ensure translations are available for lazy-loaded payloads

Fixes #20520 This should also fix #19752.

#### Changes proposed in this Pull Request:
Inline translations before enqueuing the main payload to make sure translations for lazy-loaded payloads are available. And also to avoid directory scanning and to make it work for WPCOM as well, the PR proposed to replace the `hash` from the main chunk file name with a query string based cache buster.

#### Jetpack product discussion
pcNPJE-gU-p2

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
#### Testing instructions:
* Open your local site with Jetpack Search subscription
* Navigate to the WP language folder `tools/docker/wordpress/wp-content/languages/plugins`
* Change the translation file name from `jetpack-fr_FR-8df5774da753f9be21ced70eecc80000.json` to `jetpack-fr_FR-0e17ef273599d7b1004fc0c6aacf4f5e.json`. The example is done for French language, and other languages are similar.
 * Ensure everything is translated on search interface, especially the marked ones.
 
![image](https://user-images.githubusercontent.com/1425433/131072161-9e29d951-a60f-4d63-b7cd-d362fed4a65b.png)
